datalab/datalab_session/data_operations/median.py

```diff
@@ -1,13 +1,11 @@
-from io import BytesIO
 import logging
-import os
 import tempfile
 
 import numpy as np
-from astropy.io import fits
+from fits2image.conversions import fits_to_jpg
```

Comment:
Don't forget to add this to the pyproject.toml file... I think you forgot to add the astropy/numpy requirements as well.